docs/README-dev.md

```diff
@@ -70,7 +71,7 @@ First, however, you'll need to get a necessary submodule initialized:
 If there are any problems with this step, run the following commands:
 
     cd modules/ui-common
-    git checkout hardy
+    git checkout narrative-dev
```

Comment:
This subrepo still needs 'hardy' - it's not the same as what's deployed. Not yet, at least.